db/re/item_db_equip.yml

```
@@ -101149,7 +101147,7 @@ Body:
       .@r = getrefine();
       bonus bLuk,1;
       bonus bHit,.@r;
-      if (BaseClass == Job_Swordman || BaseClass == Job_Thief || (BaseClass == Job_Taekwon && BaseJob != Job_Soul_Linker)) {
+      if (BaseClass == Job_Swordman || BaseClass == Job_Thief || (BaseClass == Job_Taekwon && BaseJob != Job_Star_Gladiator)) {
```

You misread, it was fine before
```
If equipped by Swordsman classes, Thief classes, Taewon Kid and Star Gladiator classes, Atk + 15.
```
```suggestion
      if (BaseClass == Job_Swordman || BaseClass == Job_Thief || (BaseClass == Job_Taekwon && BaseJob != Job_Soul_Linker)) {
```

---

Thanks